.github/workflows/deploy-with-docker.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         run: |
           sed -i 's/127.0.0.1:2379/172.16.238.10:2379/' api/conf/conf.yaml
           sed -i 's/host: 127.0.0.1/host: 0.0.0.0/' api/conf/conf.yaml
+          sed -i '/172.16.238.11:2379/d' api/conf/conf.yaml
+          sed -i '/172.16.238.12:2379/d' api/conf/conf.yaml
```

Comment:
@Jaycean IMHO, there is no need to remove the two lines here, as each job will be running in independent environment and configurations will not be affected by others.

@nic-chen what's your opinions?

Reply:
@imjoey  sorry, I don't get the point.. why we should change `deploy-with-docker.yml`, it's not related to this pr.

Reply:
@nic-chen This modification was added before(d07c1307ae75153d360749468091ef1e8e7bc514). @imjoey  didn't think it was necessary to add it, but it has been removed after discussing with you.

Reply:
> @imjoey sorry, I don't get the point.. why we should change `deploy-with-docker.yml`, it's not related to this pr.

@nic-chen 😄 just a misunderstanding. We both consider `deploy-with-docker.yml` should not be changed.